tests/attention/conftest.py

```
@@ -0,0 +1,104 @@
+    specs = {}
+    for it in xqa_items:
+        cs = getattr(it, "callspec", None)
+        if cs is None:
+            continue
+        kwargs = _executed_xqa_uri(cs.params)
+        if kwargs is None:
+            continue
+        try:
+            spec = gen_xqa_module(**kwargs)
+        except Exception:
+            continue  # unsupported config the test would also skip
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)